client/components/layout/managePool.tsx

```diff
@@ -0,0 +1,34 @@
+import Album from "@/types/albumTypes";
+import Artist from "@/types/artistTypes";
+import Playlist from "@/types/playlistTypes";
+import Track from "@/types/trackTypes";
+import { Box, Container, Stack, Typography } from "@mui/material";
+import TrackCard from "./cards/trackCard";
+import PoolCard from "./cards/poolCard";
+import { useEffect } from "react";
+
+export default function ManagePool(props: {
+    pool: Array<Album | Track | Artist | Playlist>
+    token: string
+    handleDelete: (itemToDelete: Album | Track | Artist | Playlist) => void
+}) {
+
+    return (
+        <Box sx={{
+            bgcolor: 'secondary.dark',
+            display: 'flex',
+            overflow: 'auto',
+            borderRadius: '12px',
```

Comment:
For future there should maybe be an agreed upon borderRadius that's used in all components

Reply:
Yes